blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnableRunTest.java

```diff
@@ -17,14 +24,18 @@
 
 import static org.mockito.Mockito.*;
 
+@PowerMockIgnore({"javax.management.*", "com.rackspacecloud.blueflood.utils.Metrics", "com.codahale.metrics.*"})
+@PrepareForTest({ IOContainer.class })
+@RunWith(PowerMockRunner.class)
+@SuppressStaticInitializationFor( "com.rackspacecloud.blueflood.io.IOContainer" )
```

Comment:
see above.  Only if we still want to test verifyZeroInteractions(locatorIO) with true mocking.